libs/expo/betterangels/src/lib/screens/NotesHmis/HmisProgramNoteEdit/HmisProgramNoteEdit.tsx

```diff
@@ -30,13 +35,115 @@ type TProps = {
   onSuccess?: () => void;
 };
 
+function splitBucket(bucket?: {
```

Comment:
this seems functionally identical to the `splitBucket` in `HmisProgramNoteCreate.tsx`. Could it be extracted to a shared util?